BuildAssets/repos/test-linux-package-feed-dists.sh

```diff
@@ -26,10 +26,10 @@ do
   docker pull "$DOCKER_IMAGE" >/dev/null || exit
   docker run --rm \
     --hostname "testfeedpkgs$RANDOM" \
-    --volume "$(pwd):/working" --volume "$SCRIPT_DIR/test-octopuscli-feed-package.sh:/test-octopuscli-feed-package.sh" \
+    --volume "$(pwd):/working" --volume "$SCRIPT_DIR/test-linux-package-feed.sh:/test-linux-package-feed.sh" \
```

Comment:
This is testing a package from the feed, right, not the feed itself?
This name feels like its testing that the feed works.

Reply:
Well, at this stage we've already tested the package file. We test it again from the feed to ensure the feed works.

Reply:
Note: I don't currently check that the feed is actually returning the new release, because we _might not_ be shipping the newest release (especially on prerelease where the exact sequence is unclear), and I'm mostly concerned with making sure a customer can still install *something*. I'd like to tighten that up at some point, although I haven't decided exactly which thing is best to assert.

Reply:
That's also why it tests `Tentacle version` still works.